ui: Statements truncated header

vladlos opened the pull request:
Statements grey header extended all the way to the right edge.

![header](https://user-images.githubusercontent.com/12850886/78563427-9e1b5000-7823-11ea-81d2-e68c3722f738.png)

Resolves: #46572

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): layout sizes update

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/46627)
<!-- Reviewable:end -->

vladlos committed:
ui: Statements truncated header

Statements grey header extended all the way to the right edge.

Resolves: #46572

Release justification: bug fixes and low-risk updates to new functionality

Release note (ui): layout sizes update

dhartunian reviewed (commented):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46627#-:-M4R-hRY0TSKbraIlzmh:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

dhartunian reviewed (approved):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/46627)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

dhartunian commented:
bors r+
